WPFWijkagent/WPFWijkagent/OffenceController.cs

```diff
@@ -23,7 +23,7 @@ public OffenceController()
         /// <param name="location">the location object to add</param>
         public void SetOffenceData(Location location)
         {
-            SetOffenceData("", OffenceCategories.categorie1, new DateTime().ToLocalTime(), location);
+            SetOffenceData("", OffenceCategories.Cybercrime, new DateTime().ToLocalTime(), location);
```

Comment:
Kan er misschien helemaal uit? Ligt er aan of iemand straks nog wat met deze methodes wil.